hack/test-integration.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -euo pipefail
+
+if [ -z "${KUBECONFIG:-}" ]; then echo "KUBECONFIG is required"; exit 1; fi
+
+# Required for the operator-sdk.
+export KUBERNETES_CONFIG="${KUBECONFIG}"
+
+go test -v -tags integration ./test/integration
```

Comment:
What are we testing? integration_test.go is empty
Are we not removing integration tests in favor of comprehensive origin/upstream e2e dns tests?

Reply:
I can imagine adding some other more focused operator-centric tests here later, didn't want to tear down all the machinery yet if it's benign.